server/server.go

```diff
@@ -306,6 +306,15 @@ func (s *Server) NewSession(ipdcID, clientID, clientState string, redirectURL ur
 	return s.SessionManager.NewSessionKey(sessionID)
 }
 
+func (s *Server) connector(id string) (connector.Connector, bool) {
+	for _, c := range s.Connectors {
```

Comment:
I know everywhere else uses the slice currently and the number of connectors in generally small, but it might be a good idea to consider adding a map[string]Connectors for an id->connector mapping. Doesn't have to be now though.